packages/unenv-preset/src/runtime/node/util/index.ts

```diff
@@ -1,51 +1,15 @@
 // https://nodejs.org/api/util.html
 import {
-	_errnoException,
-	_exceptionWithHostPort,
 	getSystemErrorMap,
 	getSystemErrorName,
-	isArray,
-	isBoolean,
-	isBuffer,
-	isDate,
-	isDeepStrictEqual,
-	isError,
-	isFunction,
-	isNull,
-	isNullOrUndefined,
-	isNumber,
-	isObject,
-	isPrimitive,
-	isRegExp,
-	isString,
-	isSymbol,
-	isUndefined,
 	parseEnv,
 	styleText,
 } from "unenv/runtime/node/util/index";
 import type nodeUtil from "node:util";
 
 export {
-	_errnoException,
-	_exceptionWithHostPort,
 	getSystemErrorMap,
 	getSystemErrorName,
-	isArray,
```

Comment:
Note: `isArray` is not removed from Node and the timeline to remove the other functions is Node 23.

Reply:
isArray is available and implemented on workerd

Reply:
I should have commented on l94 instead.
What I meant is that `isArray` is not part of the functions removed in Node23.

But if we have some of the functions here implemented in workerd (and not going away) we should definitely use the workerd version instead - I guess it will be a different PR.

Thanks